src/explore-education-statistics-admin/src/pages/release/data/components/DraftApiDataSetsTable.tsx

```diff
@@ -20,17 +20,21 @@ import React from 'react';
 import { generatePath } from 'react-router-dom';
 import styles from './DraftApiDataSetsTable.module.scss';
 
+export const columnWidth = 90;
```

Comment:
Do we tend to set fixed pixel widths in other places in the UI?

Reply:
We generally try to avoid it but occasionally it's necessary. In this case it's needed to get the columns to align in the live and draft tables and since we know the content of these columns is always a Tag with a version number they don't need to be flexible.

Reply:
thanks